pageserver/src/tenant/mgr.rs

```diff
@@ -181,25 +183,11 @@ pub async fn shutdown_all_tenants() {
         tenants_to_shut_down
     };
```

Comment:
The HTTP endpoint is still active, I think. So, theoretically, a tenant_attach request could come in, and we'd have the tenant active twice.
Probably not a regression compared to what we had before, but my hope was that RequestContext would take care of such race conditions.

Perhaps by re-ordering the shutdown sequences in the caller, plus preventing new TenantRequestContext's from being created?